components/n-ui/ads/index.js

```diff
@@ -22,6 +22,36 @@ function addZone (content) {
 	}
 }
 
+const contentIsV2 = content => !(content && content.dfp);
```

Comment:
It's a personal preference, but this function would be a bit easier to read if it was written like `contentIsV1`. Negations always make it a bit harder to read.